### PR TITLE
Add CREATE VIEW

### LINKED ISSUE #380: Document the support for CREATE VIEW command
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/6023

### Which part(s) of the docs might be affected or should be updated? And how?

Add a new topic for CREATE VIEW

### Reference

See e2e_test/batch/basic/logical_view.slt.part for examples. 
https://www.postgresql.org/docs/current/sql-createview.html

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

Preview: https://pr-405.d2fbku9n2b6wde.amplifyapp.com/docs/latest/sql-create-view/

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/6023

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/380

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Review by xxchan — APPROVED** [review]
LGTM

File: docs/sql/commands/sql-create-view.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+|Parameter                  | Description           |
+|---------------------------|-----------------------|
+|*mv_name*                  |The name of the view to be created.|
+|*column_name*              |Specify the columns of the view.|
+|*select_query*             |A SELECT query that retrieves data for the view. See [SELECT](sql-select.md) for the syntax and examples of the `SELECT` command.|
+
+
+## Examples
+
+The following statement shows how to create views based on a user-defined table and a materialized source (whose data is generated by the built-in [load generator](../create-source/create-source-datagen.md)), and then create a new view based on the existing views.
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-view.md]: @CharlieSYH Althought these statements are wrapped in one code block, they are still separate statements. I would rephrase this sentences a bit, to something like: The following statements create views based a user-defined table and a materialized source, and then create a new view based on the existing views. The data for the materialized source is generated by the built-in ...

**Review by hengm3467 — APPROVED** [review]
I left one comment. The rest looks good to me.
